docs/api/guides/guide-to-creating-an-api.md

```diff
@@ -0,0 +1,324 @@
+---
+sidebar_position: 5
+---
+
+# Creating an API
+###### These are the various steps that are involved in creating a new endpoint.
+
+:::note
+This is assumed you are creating a new API router on an existing server, this documentation will not cover how to create a new server. 
+:::
+Prior to creating a new API, we must first implement the swagger definition of said new API. We are using Swagger 2.0, which is [OpenAPI](https://swagger.io/specification/v2/), a specification we use to format our RESTful APIs and provide a template for us to communicate the information in our API.
+Always start with swagger. This step creates your API's endpoint definitions and generates the files and helper functions you will need to create your endpoint. More specifically, swagger converts JSON user input into generated Go types.
+
+## Creating a new API yaml
+
+All new API's require a new yaml file to be added in `mymove/swagger-def`. This directory holds many different specs and definitions as they get reused throughout the application.
+You will notice that there is also a `mymove/swagger` folder which holds what looks like to be a copy of `swagger-def`, please do not edit within this folder. The `swagger` folder
```

Comment:
which holds what looks like to be a copy

I think this has an extra word in there, maybe

which holds what looks to be a copy

Reply:
Thank you, adjusted now